jest/setupJestMock.js

```diff
@@ -13,9 +13,48 @@ global.SimVar.GetSimVarValue = jest.fn((name, _, __) => {
         return 0;
     }
 });
-global.SimVar.SetSimVarValue = jest.fn((name, _, value, __) => {
+
+global.SimVar.SetSimVarValue = jest.fn((name, _, value) => {
     return new Promise((resolve, _) => {
         values[name] = value;
         resolve();
     });
 });
+
+let listeners = {};
+
+global.beforeEach(() => {
+    listeners = {};
+});
+
+/** @type {Partial<ViewListener.ViewListener>} */
+const MockViewListener = {
```

Comment:
**question**: Is the code duplicate intentional?

Reply:
No. The TS file wasn't meant to be committed.